### PR TITLE
[TT-9769] Rename helm charts

### PR DESCRIPTION
**Pull request by komalsukhani**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Preview Link

[Self managed](https://deploy-preview-3183--tyk-docs.netlify.app/docs/nightly/tyk-self-managed/tyk-helm-chart-tyk-stack/)
[Tyk Cloud](https://deploy-preview-3183--tyk-docs.netlify.app/docs/nightly/tyk-cloud/environments-deployments/hybrid-gateways/#deploy-in-kubernetes-with-helm)

### Description

**1. Renamed helm charts as follow**

- tyk-stack (formerly called tyk-single-dc)

- tyk-control-plane (formerly called tyk-mdcb-control-plane)

- tyk-data-plane (formerly called tyk-mdcb-data-plane)

**2. Renamed `tyk-self-managed/tyk-helm-chart-single-dc.md` to `tyk-self-managed/tyk-helm-chart-tyk-stack.md`**

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR looks good, however there is a broken link reported in the CI checks. If look in the output this references `tyk-self-managed/tyk-helm-chart-single-dc`

It could be that this is referenced somewhere else still after being removed from the menu.yaml?

**Review by carolwkc — APPROVED** [review]

File: tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ description: "How to deploy Tyk Self Managed on Kubernetes using new Helm Chart"
 menu:
   main:
     parent: "Tyk Helm Chart "
+aliases:
```
Comment (letzya) [inline tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md]: @dcs3spp we cant just rename a url, we need to move it to the appropriate location.

File: tyk-docs/data/menu.yaml [hunk 2]
```diff
@@ -441,7 +441,7 @@ menu:
               category: Page
               show: True
             - title: "Tyk new Helm chart"
```
Comment (letzya) [inline tyk-docs/data/menu.yaml:439]: @caroltyk are you sure about the "New"? shouldn't we add a message that it's new, in the body of the page? and in the release notes?

Also, the title doesn't fit the format we used on the page of the old charts. Can we maybe make it more consistent, also can use "old" or "deprecated" on the old ones and remove the "new" here?

**Review by letzya — CHANGES_REQUESTED** [review]
release label is missing  + other comments in the PR

File: tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md (hunk 1 above)
Comment (komalsukhani) [reply tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md]: Sorry. Will fix it.

**Comment by letzya** [comment]
@dcs3spp flagging out an issue with renaming a file

File: tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways-helm.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ weight: 5
 {{< warning success >}}
 **Warning**
 
-`tyk-hybrid` will be deprecated soon. Please use our new Helm Chart for Tyk Hybrid Data Plane at [tyk-mdcb-data-plane]({{<ref "/tyk-cloud/environments-deployments/hybrid-gateways">}}) instead. 
+`tyk-hybrid` will be deprecated soon. Please use our new Helm Chart for Tyk Hybrid Data Plane at [tyk-data-plane]({{<ref "/tyk-cloud/environments-deployments/hybrid-gateways">}}) instead.
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways-helm.md]:
```suggestion
`tyk-hybrid` will be deprecated soon. Please use our new Helm Chart for Tyk Hybrid Data Plane at [tyk-data-plane]({{<ref "tyk-cloud/environments-deployments/hybrid-gateways">}}) instead. 
```

File: tyk-docs/content/deployment-and-operations/tyk-self-managed/deployment-lifecycle/installations/kubernetes/tyk-helm-tyk-stack.md [hunk 4]
```diff
@@ -19,11 +15,11 @@ Tyk is working to provide a new set of helm charts, and will progressively roll
 The new Helm Charts are in beta stage. Breaking changes may be introduced before stable release.
 {{< /warning >}}
 
-To deploy Tyk Self Managed (for single data center) using the new helm chart, please use [tyk-single-dc](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-single-dc) chart.
+To deploy Tyk Self Managed (for single data center) using the new helm chart, please use [tyk-stack](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-stack) chart.
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-self-managed/deployment-lifecycle/installations/kubernetes/tyk-helm-tyk-stack.md:21]: @caroltyk why don't we send them to artifact hub? and give the GH link as a side note?

File: tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/content/tyk-self-managed/tyk-helm-chart-tyk-stack.md]: Fixed and moved to tyk-docs/content/deployment-and-operations/tyk-self-managed/deployment-lifecycle/installations/kubernetes/tyk-helm-tyk-stack.md

File: tyk-docs/data/menu.yaml (hunk 2 above)
Comment (dcs3spp) [reply tyk-docs/data/menu.yaml:439]: @letzya as discussed resolving the comment and can raise a new PR later

File: tyk-docs/content/deployment-and-operations/tyk-self-managed/deployment-lifecycle/installations/kubernetes/tyk-helm-tyk-stack.md (hunk 4 above)
Comment (carolwkc) [reply tyk-docs/content/deployment-and-operations/tyk-self-managed/deployment-lifecycle/installations/kubernetes/tyk-helm-tyk-stack.md:21]: @letzya We have tracking on GH so we could get more information there, don't we?

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**